### PR TITLE
feat: add ItemGetTtl operation

### PR DESCRIPTION
**Pull request by pgautier404**

### CODE REVIEW

File: src/Momento.Sdk/Responses/CacheItemGetTtlResponse.cs [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            value = response.Found.RemainingTtlMillis;
+        }
+
+        /// <summary>
+        /// The value of the ttl.
+        /// </summary>
+        public ulong Value
+        {
+            get => value;
+        }
```
Comment (malandis) [inline src/Momento.Sdk/Responses/CacheItemGetTtlResponse.cs:59]: Can we expose this as a `TimeSpan` object instead? That is the .NET native type which has all the ergonomic accessors.

File: tests/Integration/Momento.Sdk.Tests/TtlTest.cs [hunk 2]
```diff
@@ -110,4 +110,50 @@ public async Task UpdateTtlAsync_KeyIsStringAndExists_IsSet()
...
+    {
+        var ttlResponse = await client.ItemGetTtlAsync(Utils.NewGuidString(), Utils.NewGuidString());
+        Assert.True(ttlResponse is CacheItemGetTtlResponse.Error, $"Unexpected response: {ttlResponse}");
+        Assert.Equal(MomentoErrorCode.NOT_FOUND_ERROR, ((CacheItemGetTtlResponse.Error)ttlResponse).ErrorCode);
+    }
+
+    [Fact]
+    public async Task ItemGetTtl_EmptyCacheNameError()
+    {
+        var ttlResponse = await client.ItemGetTtlAsync(null, Utils.NewGuidString());
```
Comment (malandis) [inline tests/Integration/Momento.Sdk.Tests/TtlTest.cs:155]:
```suggestion
        var ttlResponse = await client.ItemGetTtlAsync(null!, Utils.NewGuidString());
```

**Review by malandis — COMMENTED** [review]
Looks great just two small changes 🙇‍♂️

File: src/Momento.Sdk/Responses/CacheItemGetTtlResponse.cs (hunk 1 above)
Comment (pgautier404) [reply src/Momento.Sdk/Responses/CacheItemGetTtlResponse.cs:59]: Yes, good catch.

**Review by malandis — APPROVED** [review]
🎉

**Review by cprice404 — APPROVED** [review]

**Pull request merged**